Jenkinsfile.release

```diff
@@ -566,11 +566,20 @@ node('zowe-jenkins-agent-dind-wdc') {
       return isFormalRelease && (releaseArtifacts['docker-amd64'] || releaseArtifacts['docker-s390x'])
     },
     stage            : {
-      withCredentials([usernamePassword(
-        credentialsId: 'ZoweDockerhub',
-        usernameVariable: 'USERNAME',
-        passwordVariable: 'PASSWORD'
-      )]) {
+      withCredentials([
+        usernamePassword(
+          credentialsId: 'ZoweDockerhub',
+          usernameVariable: 'USERNAME',
+          passwordVariable: 'PASSWORD'
+        ),
+        sshUserPrivateKey(
+          credentialsId: 'DockerUserSigningKey',
+          keyFileVariable: 'USERSIGNKEY',
+          passphraseVariable: 'SIGNPASSPHRASE',
+          usernameVariable: 'SIGNKEYNAME'
+        )
+      ]) {
+        sh "echo \"${USERSIGNKEY}\" > ~/.docker/trust/private/${SIGNKEYNAME}.key"
```

Comment:
the keyname appears to be important for docker to be able to find the right key to use, so keeping the name intact is important, as is the location of the file.